core/src/main/java/org/nervos/ckb/type/BlockResponse.java

```diff
@@ -0,0 +1,16 @@
+package org.nervos.ckb.type;
+
+import java.util.List;
+
+public class BlockResponse {
```

Comment:
suggest to rename this class to `BlockWithCycles`

Reply:
@liuck8080 this comment was marked as resolved, however, the class name is unchanged, did you forget to push the new commit?

Reply:
Now the commit is committed